src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1351,16 +1351,16 @@ get_test_name(const std::string& input_name)
  */
 static void
 pretty_print_test_desc(const boost::property_tree::ptree& test, int& test_idx,
-                       std::ostream & _ostream, const std::string& bdf)
+                       std::ostream & _ostream, const std::string& bdf, bool single_case)
```

Comment:
As Stephen mentioned, the usage of the single_case variable here isn't ideal and contributed to why I closed the initial PR I made. I adhere to his review for his proposed changes.

Reply:
Addressed Stephen's comments and submitted changes in PR